staging/src/k8s.io/client-go/openapi3/root.go

```diff
@@ -170,3 +170,13 @@ func pathToGroupVersion(path string) (schema.GroupVersion, error) {
 	}
 	return gv, nil
 }
+
+// Encapsulates GroupVersion not found as one of the paths
+// at OpenAPI V3 endpoint.
+type GroupVersionNotFoundError struct {
+	gv schema.GroupVersion
```

Comment:
should this be public?

Reply:
If others outside the package want to check the type, they would have to cast to `GroupVersionNotFoundError`; so I think it has to be public. Unless we create and export a `IsGroupVersionNotFoundError` function in this package.

Reply:
I wasn't referring to the type, but to the gv variable